### DISCUSSION TITLE
Kitty Terminal Not Opening

### DISCUSSION THREAD

**Discussion by AKA-Akhil** [prasanthrangan/hyprdots#1465] [Q&A] [answered]
My terminal is not opening 
Super+T doesn't work
i cant even open it through rofi

**Comment by dieBakterie** [comment]
Try it over dolphins menu "Open terminal in here", some more Infos would be helpful.

**Reply by AKA-Akhil** [reply]
doesn't work there as well

**Comment by kRHYME7** [comment, marked as answer]
Change to any tty ` ctrl + alt + f4 ` 

then try to check if kitty is installed or something or install another terminal emulator inside tty so we can debug later ` yay -Sy alacritty `

**Reply by AKA-Akhil** [reply]
alacritty works

i also tried reinstalling kitty which solved the issue thanks!
